core/ops_metrics.rs

```diff
@@ -116,76 +113,3 @@ impl OpMetricsSummary {
     self.ops_dispatched_async > self.ops_completed_async
   }
 }
-
-#[derive(Default, Debug)]
-pub struct OpMetricsSummaryTracker {
```

Comment:
IIUC, `core.metrics()` was the only symbol that used this struct.

Reply:
I believe this is actually used by the `deno test` for leaking ops discovery. This should most likely stay. See `runtime/worker.rs` in Deno repo.